keps/sig-network/2595-expanded-dns-config/README.md

```diff
@@ -103,6 +103,14 @@ DNS search path list to an arbitrary number
 
 ### Notes/Constraints/Caveats (Optional)
 
+Some container runtimes of older versions have their own restrictions on the
+number of DNS search paths. For the container runtimes which are older than
+- Containerd v1.6
+- CRI-O v1.22
```

Comment:
@mrunalp same question as above but for crio 
https://github.com/kubernetes/enhancements/pull/2884/files#r692976930

Reply:
https://github.com/cri-o/cri-o#compatibility-matrix-cri-o--kubernetes
> If a Kubernetes release goes End of Life, then the corresponding CRI-O version can be considered in the same way.

https://kubernetes.io/releases/#release-history

> kubernetes v1.21, End of Life: 2022-06-28